src/components/MenuBar/MenuBar.tsx

```diff
@@ -81,38 +84,51 @@ export default function MenuBar() {
   const reporterToken: string = window.location.hash.substr(1);
   let repoterInfo: any = reporterToken ? jwt_decode(reporterToken) : '';
   const classes = useStyles();
-
-  const { getToken, isFetching } = useAppState();
+  const { setError, getToken, isFetching } = useAppState();
```

Comment:
please see if we have a way to display messages without using setError e.g. something like setInfo